app/assets/stylesheets/cart.css.scss

```diff
@@ -30,11 +30,12 @@
 }
 
 #cart-table {
-    margin-left:50px;
-    width: 850px;
-    border: none;
-    border-radius: 5px;
-    box-shadow: 0px 10px 20px rgba(0,0,0,.3);
+  margin-left:50px;
```

Comment:
Properties should be ordered border, border-radius, box-shadow, margin-left, width<br>Colon after property should be followed by one space